spec/models/article_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe Article, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :title }
+    it { is_expected.to have_db_column :sub_title }
+    it { is_expected.to have_db_column :content}
+  end
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :title }
+    it { is_expected.to validate_presence_of :content}
+  end
+  describe 'Factory' do
+    it 'should have valid Factory' do
+      expect(FactoryBot.create(:article)).to be_valid
```

Comment:
You can omit `FactoryBot` if you have setup your `rails_helper` properly

Reply:
Deleted it and checked the test goes green.